proto/ibc/applications/transfer/v1/authz.proto

```diff
@@ -22,6 +23,14 @@ message Allocation {
   // allow list of memo strings, an empty list prohibits all memo strings;
   // a list only with "*" permits any memo string
   repeated string allowed_packet_data = 5;
+  // a list of Allowed hops
+  repeated Hops allowed_forwarding_hops = 6;
+}
+
+
+// Hops represent a list of Hop
+message Hops {
```

Comment:
Is there any specific reason to introduce a new type here wrapping `Hop`? couldn't we use a `repeated Hop` field within the wrapper inside of `Allocation`?

Reply:
I'm not sure I follow: `allowed_forwarding_hops` is a list of list of Hops. So we can't replace it with a `repeated Hop`. It is a `repeated repeated Hop`.

But let me know if I didn't get your comment!